x-pack/plugins/integration_assistant/server/graphs/kv/error.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { JsonOutputParser } from '@langchain/core/output_parsers';
+import type { KVState } from '../../types';
+import type { HandleKVNodeParams } from './types';
+import { KV_ERROR_PROMPT } from './prompts';
+import { COMMON_ERRORS, KV_EXAMPLE_ANSWER } from './constants';
+import { createKVProcessor } from '../../util/processors';
+import { KVProcessor } from '../../processor_types';
+
+export async function handleKVError({
+  state,
+  model,
+}: HandleKVNodeParams): Promise<Partial<KVState>> {
+  const outputParser = new JsonOutputParser();
+  const kvHeaderGraph = KV_ERROR_PROMPT.pipe(model).pipe(outputParser);
+  const proc = state.kvProcessor;
+  const processor = proc[0].kv;
+
+  const kvInput = (await kvHeaderGraph.invoke({
+    current_processor: JSON.stringify(processor, null, 2),
+    errors: JSON.stringify(state.errors, null, 2),
+    common_errors: JSON.stringify(COMMON_ERRORS, null, 2),
+    ex_answer: JSON.stringify(KV_EXAMPLE_ANSWER, null, 2),
+  })) as KVProcessor;
+
+  const kvProcessor = createKVProcessor(kvInput, state);
+
+  return {
+    kvProcessor,
+    lastExecutedChain: 'kv_header',
```

Comment:
don't we want this to be `kv_error`, or something like that to indicate the execution of the error prompt?